examples/resources/datadog_domain_allowlist/resource.tf

```diff
@@ -0,0 +1,4 @@
+resource "datadog_ip_allowlist" "example" {
+  enabled = true
```

Comment:
So that should be able to be replicated within the tests. I updated to set that in [line 72 in the test file](https://github.com/DataDog/terraform-provider-datadog/pull/2637/files#diff-a1ee7405e74db778a756cf25fea3ba5e402403a9a244f9381893d1eb5366e443R72)

The error message from the command is
```
=== RUN   TestAccDatadogDomainAllowlist_CreateUpdate
    resource_datadog_domain_allowlist_test.go:21: Step 1/2 error: Error running pre-apply refresh: exit status 1
        
        Error: Invalid reference
        
          on terraform_plugin_test.tf line 3, in resource "datadog_domain_allowlist" "foo":
           3: 	enabled = False
        
        A reference to a resource type must be followed by at least one attribute
        access, specifying the resource name.
```
I'm not sure if that's the general command we return when the input is invalid in the provider.

If you want to confirm the command to run it against backend instead of the saved cassette file is `TESTARGS="-run TestAccDatadogDomainAllowlist_CreateUpdate" make cassettes`